ludwig/contribs/mlflow/__init__.py

```diff
@@ -21,14 +21,21 @@ def _get_runs(experiment_id: str):
     return mlflow.tracking.client.MlflowClient().search_runs([experiment_id])
 
 
-def _get_or_create_experiment_id(experiment_name, artifact_uri: str = None):
+@DeveloperAPI
+def get_or_create_experiment_id(experiment_name, artifact_uri: str = None):
+    """Gets experiment id from mlflow."""
     experiment = mlflow.get_experiment_by_name(experiment_name)
     if experiment is not None:
         return experiment.experiment_id
     return mlflow.create_experiment(name=experiment_name, artifact_location=artifact_uri)
 
 
-@DeveloperAPI
+# Included for backwards compatibility, Deprecated.
+# TODO(daniel): delete this.
+_get_or_create_experiment_id = get_or_create_experiment_id
```

Comment:
Should we add DeprecatedAPI to this?

Reply:
Unfortunately decorators don't work for global variables, only for function-like objects (functions, classes, enums).  This will be a problem too for constants.py or globals.py.